### PR TITLE
Test cases for lang_is_a module

### PR DESCRIPTION
**Pull request by kaviyachandran**
This PR has test cases for module lang_is_a.

### CODE REVIEW

File: src/lang/terms/is_a.pl [hunk 1]
```diff
@@ -142,3 +142,43 @@
...
+  assert_true(ask(instance_of_all('Greta', [test:'Woman', dul:'Person']))),
+  assert_true(ask(instance_of('Greta', [test:'Woman', dul:'Person']))).
+
+test("ask and tell sub property of") :-
+  assert_true(subproperty_of(test:'hasParent', test:'hasAncestor')),
+  assert_false(subproperty_of(test:'hasBrother', test:'hasSibling')),
+  assert_true(tell(subproperty_of(test:'hasBrother', test:'hasSibling'))),
+  assert_true(subproperty_of(test:'hasBrother', test:'hasSibling')).
+
+test("ask A is a Person and expect instantiation error as is_a expects a ground variable") :-
```
Comment (daniel86) [inline src/lang/terms/is_a.pl]: better try to have a bit more compact name for the testcase.
If you think needed, add a comment in addition.

File: src/lang/terms/is_a.pl [hunk 2]
```diff
@@ -142,3 +142,43 @@
...
+  catch(is_a(A, dul:'Person'), error(E, _), true),
```
Comment (daniel86) [inline src/lang/terms/is_a.pl]: cool. But plunit provides an option for testcases that throw errors.
Have a look for the *throws* option: https://www.swi-prolog.org/pldoc/doc_for?object=section(%27packages/plunit.html%27)

**Review by daniel86 — CHANGES_REQUESTED** [review]

File: src/lang/terms/is_a.pl (hunk 1 above)
Comment (kaviyachandran) [reply src/lang/terms/is_a.pl]: Sure

File: src/lang/terms/is_a.pl (hunk 2 above)
Comment (kaviyachandran) [reply src/lang/terms/is_a.pl]: I will use throws. Did you check the line number 67? Is the change okay

Comment (daniel86) [reply src/lang/terms/is_a.pl]: yes I have seen that. Best would be to do address this in a separate pull request.
But well, it's ok, I guess

**Review by daniel86 — APPROVED** [review]

**Pull request merged**